<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblDate">
    <property name="geometry">
     <rect>
      <x>679</x>
      <y>80</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnMeteo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>123</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Changer de Ville</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextMeteo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>173</y>
      <width>251</width>
      <height>481</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="documentTitle">
     <string/>
    </property>
    <property name="backgroundVisible">
     <bool>false</bool>
    </property>
    <property name="centerOnScroll">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbliconmeteo">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>393</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>J</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblHeure">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>123</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Heure</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnPrevisions">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>360</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Affichage Pr√©visions</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextMer">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>173</y>
      <width>251</width>
      <height>481</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="documentTitle">
     <string/>
    </property>
    <property name="backgroundVisible">
     <bool>false</bool>
    </property>
    <property name="centerOnScroll">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Cadre_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>670</y>
      <width>1001</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 127);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="documentTitle">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="backgroundVisible">
     <bool>false</bool>
    </property>
    <property name="centerOnScroll">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Cadre_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 127);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="documentTitle">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="backgroundVisible">
     <bool>false</bool>
    </property>
    <property name="centerOnScroll">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Cadre_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>61</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 127);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="documentTitle">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="backgroundVisible">
     <bool>false</bool>
    </property>
    <property name="centerOnScroll">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Cadre_1">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>60</y>
      <width>61</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 127);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="documentTitle">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="backgroundVisible">
     <bool>false</bool>
    </property>
    <property name="centerOnScroll">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelVille">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>93</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ville : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVille">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>93</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Paris</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextPrevisions">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>73</y>
      <width>321</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="documentTitle">
     <string/>
    </property>
    <property name="backgroundVisible">
     <bool>false</bool>
    </property>
    <property name="centerOnScroll">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="iconeJ1">
    <property name="geometry">
     <rect>
      <x>589</x>
      <y>93</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>J+1</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="iconeJ2">
    <property name="geometry">
     <rect>
      <x>588</x>
      <y>208</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>J+2</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="iconeJ3">
    <property name="geometry">
     <rect>
      <x>588</x>
      <y>323</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>J+3</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="iconeJ4">
    <property name="geometry">
     <rect>
      <x>588</x>
      <y>433</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>J+4</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="iconeJ5">
    <property name="geometry">
     <rect>
      <x>588</x>
      <y>550</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>J+5</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>plainTextPrevisions</zorder>
   <zorder>iconeJ1</zorder>
   <zorder>iconeJ2</zorder>
   <zorder>iconeJ3</zorder>
   <zorder>iconeJ4</zorder>
   <zorder>iconeJ5</zorder>
   <zorder>plainTextMer</zorder>
   <zorder>plainTextMeteo</zorder>
   <zorder>lblDate</zorder>
   <zorder>BtnMeteo</zorder>
   <zorder>lbliconmeteo</zorder>
   <zorder>lblHeure</zorder>
   <zorder>BtnPrevisions</zorder>
   <zorder>Cadre_2</zorder>
   <zorder>Cadre_4</zorder>
   <zorder>Cadre_3</zorder>
   <zorder>labelVille</zorder>
   <zorder>lineEditVille</zorder>
   <zorder>Cadre_1</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>&amp;Administration</string>
    </property>
    <addaction name="action_Administration"/>
    <addaction name="action_Quitter"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <action name="action_Administration">
   <property name="text">
    <string>&amp;Param√®tres</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Quitter">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
